### PR TITLE
Add code snippets for editor examples

### PR DESCRIPTION
**Pull request by godai78**
### Suggested merge commit message ([convention](https://ckeditor.com/docs/ckeditor5/latest/framework/guides/contributing/git-commit-message-convention.html))

Docs: Add code snippets for full-featured and bottom toolbar editor examples

---

### Additional information

Part of: #11622

Merge together with: https://github.com/cksource/ckeditor5-internal/pull/1691

### CODE REVIEW

File: docs/examples/custom/bottom-toolbar-editor.md [hunk 1]
```diff
@@ -14,4 +15,422 @@ Additionally, thanks to the flexibility offered by the {@link framework/guides/a
...
+
+		window.attachTourBalloon( {
+			target: window.findToolbarItem( editor.ui.view.toolbar,
+				item => item.label && item.label === 'Formatting options' ),
+			text: 'Click to open formatting options.',
+			editor,
+			tippyOptions: {
+				placement: 'bottom-start'
+			}
+		} );
```
Comment (FilipTokarski) [inline docs/examples/custom/bottom-toolbar-editor.md:283]: We don't need the tour balloon.

File: docs/examples/custom/bottom-toolbar-editor.md [hunk 2]
```diff
@@ -14,4 +15,422 @@ Additionally, thanks to the flexibility offered by the {@link framework/guides/a
...
+
+## Editor example configuration
+
+<details>
+<summary>View editor configuration script</summary>
+
+```js
+
+/* globals console, window, document */
+
```
Comment (FilipTokarski) [inline docs/examples/custom/bottom-toolbar-editor.md:30]: We can remove those lines in both snippets as well.

**Review by FilipTokarski — APPROVED** [review]
Lets remove those lines I commented and I think we're good to go here 👍

File: docs/examples/custom/bottom-toolbar-editor.md (hunk 1 above)
Comment (godai78) [reply docs/examples/custom/bottom-toolbar-editor.md]: Removed.

File: docs/examples/custom/bottom-toolbar-editor.md (hunk 2 above)
Comment (godai78) [reply docs/examples/custom/bottom-toolbar-editor.md]: Removed in both.

**Pull request merged**